recipe/meta.yaml

```
@@ -147,6 +147,7 @@ outputs:
         - llvm-openmp  # [osx]
       host:
     {% if cuda_major == 12 %}
+        - cuda-version {{ cuda_compiler_version }}
```

It might be worth doing this generally

```suggestion
        - cuda-version {{ cuda_compiler_version }}
    {% if cuda_major == 12 %}
```

---

NVCC 11.2 doesn't care that it's linking against libraries from 11.8. Only NVCC 12.0 cares if it's linking against libraries from 12.1. The nvcc package should maybe have host_exports (if that existed).